### PR TITLE
3.4.5 Feature/empty avatar styles

### PR DESCRIPTION
**Pull request by schweitzdesign**
Adds new styles for default Initial Avatars shared between mars and store.

screen shot 2018-04-26 at 12 43 53 pm

### CODE REVIEW

**Comment by schweitzdesign** [comment]
Easiest to run this against https://github.com/forever-inc/a-new-hope/pull/1366 which has all the store side logic

**Comment by schweitzdesign** [comment]
Please use this guy instead for reference: https://github.com/forever-inc/a-new-hope/pull/1371

File: app/assets/stylesheets/forever_style_guide/modules/_avatar.scss [hunk 1]
```diff
@@ -1,35 +1,80 @@
...
+$avatar-lg: 150px;
+$avatar-md: 100px;
+$avatar-sm: 70px;
+$avatar-xs: 40px;
+$avatar-shadow: 0 5px 15px rgba(0, 0, 0, 0.2);
+
+@mixin avatar_size($size) {
+  height: $size;
+  width: $size;
+  min-width: $size; // dont' squish my flex, man
```
Comment (alexisBimatovs) [inline app/assets/stylesheets/forever_style_guide/modules/_avatar.scss]: 💪 🏋️‍♀️

**Comment by alexisBimatovs** [comment]
screen shot 2018-04-27 at 2 04 00 pm
screen shot 2018-04-27 at 2 05 38 pm
screen shot 2018-04-27 at 2 02 19 pm

**Comment by alexisBimatovs** [comment]
LGTM!

**Pull request merged**